src/axom/quest/Delaunay.hpp

```diff
@@ -395,20 +395,22 @@ class Delaunay
     return valid;
   }
 
```

Comment:
Sure -- I can do that!

I've been using the vtk file to look at the triangulation, but the (recently renamed) `_input` mesh to look at the fields.
It should be easy enough to add the fields directly to the vtk file.

Reply:
Thanks!

Reply:
Thanks for the suggestion -- the vtk output now includes the scalar fields*. 

I also converted the vtk output into a user-callable function (rather than automatically getting called as part of the `buildTriangulation()` step).

* I only added the scalar fields of type `float64`, since other types seemed to not be of interest. I can add others as necessary/desired.

Reply:
Looks good to me, thanks!